blazor/datagrid/grouping.md

```diff
@@ -296,4 +296,60 @@ The following sample code demonstrates the above,
 The following screenshot represents a DataGrid with customized group caption,
 ![Blazor DataGrid with Custom Group Caption](./images/blazor-datagrid-custom-group-caption.png)
 
+## Group column programmatically
+
+The Grid's [GroupColumnAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_GroupColumnAsync_System_String_) and [UngroupColumnAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_UngroupColumnAsync_System_String_) methods can be used to group and ungroup columns programmatically.
```

Comment:
As i explained earlier, here you didnt mention that the dynamic grouping and ungrouping is done using button and how its works by default

Reply:
Changed the content as suggested.